### PR TITLE
[RDY] Suggestion - Give nurse dynamic info when vaccinating

### PR DESCRIPTION
**Pull request by lewri**
Addition to #1620 that adds an improvement for when the nurse is assigned a patient to vaccinate.

Currently, the nurse has the dynamic info "Just wandering around" when going to vaccinate a patient.
This PR would change her dynamic info to "Vaccinating a patient" in that time frame.

Dyanmic info resets back to normal once vaccination is done.

### CODE REVIEW

File: CorsixTH/Lua/languages/english.lua [hunk 1]
```diff
@@ -260,6 +260,7 @@ dynamic_info.patient.actions.no_gp_available = "Waiting for you to build a GP's
 dynamic_info.staff.actions.heading_for = "Heading for %s"
 dynamic_info.staff.actions.fired = "Fired"
 dynamic_info.patient.actions.epidemic_vaccinated = "I am no longer contagious"
+dynamic_info.patient.actions.vaccine = "Vaccinating a patient"
```
Comment (mugmuggy) [inline CorsixTH/Lua/languages/english.lua:263]: staff not patient

Comment (lewri) [reply CorsixTH/Lua/languages/english.lua:263]: Good spot, thanks!

**Comment by TheCycoONE** [comment]
Does this get cleared properly after the patient is vaccinated (or the action is interrupted)?

**Comment by lewri** [comment]
I'm not entirely sure. And I wouldn't know what to really check.
All I can tell you is after the patient is vaccinated it goes back to 'Just wandering around'.
I can also tell you if the nurse is mid-action and the patient starts moving, the "giveVaccinationCandidateStatus" is stopped and goes back to 'Just wandering around'.

If that's not what you're asking I'm not sure, sorry.

**Pull request merged**